ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/PeerSelection.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections       #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Ouroboros.Consensus.Shelley.Ledger.PeerSelection () where
```

Comment:
This module exports only the orphaned instance.   Is there a central place in `ouroboros-consensus-shelley` which imports all orphaned instance modules (if there are other ohpans)?

Reply:
Yes, see my other comment "Here".

Reply:
:+1: